automation/test.sh

```diff
@@ -7,6 +7,12 @@ export GOPATH=$(pwd)/go
 export PATH=$GOPATH/bin:$GOROOT/bin:$PATH
 mkdir -p $GOPATH
 
+echo "Install Go 1.13.5"
```

Comment:
this can be deleted, it's done below now

Reply:
oh, this should have been the 1st comment of a review...

Reply:
done, removed duplicated installation (I ommited to review an automatic merge)